lib/tasks/check_content_consistency.rake

```diff
@@ -4,8 +4,14 @@ namespace :check_content_consistency do
     errors = checker.call
 
     if errors.any?
-      puts "#{content_id} #{locale} 😱"
-      puts errors
+      Airbrake.notify(
+        "Found an inconsistent document: #{content_id} #{locale} 😱",
```

Comment:
Not a 100% sure you can pass it a string, I remember vaguely that it has to be an exception object. Could be wrong though.

In any case, if you use the same message/string for all errors they'll be rolled up into a single exception. Using a message per-error might clutter errbit and cause an overload of emails, which you may not want.